### PR TITLE
[WIP][v2] Edit tutorial to use emotion instead of glamorous

### PR DESCRIPTION
**Pull request by ryanditjia**
Refs: https://github.com/gatsbyjs/gatsby/issues/5607

[Working repo of tutorial](https://github.com/ryanditjia/gatsby-v2-tutorial-emotion) (see branches).

### CODE REVIEW

File: docs/tutorial/part-four/index.md [hunk 1]
```diff
@@ -132,35 +133,38 @@ export default () => (
...
 import React from "react"
-import g from "glamorous"
-import { css } from "glamor"
+import { css } from "react-emotion"
 import { Link } from "gatsby"
 
 import { rhythm } from "../utils/typography"
 
-const linkStyle = css({ float: `right` })
+const container = css`
```
Comment (KyleAMathews) [inline docs/tutorial/part-four/index.md]: Can you move these to inline in their components please

File: docs/tutorial/part-four/index.md [hunk 2]
```diff
@@ -270,15 +272,29 @@ export const query = graphql`
...
 import React from "react"
-import g from "glamorous"
-import { css } from "glamor"
+import { css } from "react-emotion"
 import { StaticQuery, Link } from "gatsby"
 
 import { rhythm } from "../utils/typography"
 
-const linkStyle = css({ float: `right` })
+const container = css`
```
Comment (KyleAMathews) [inline docs/tutorial/part-four/index.md]: same here

Comment (KyleAMathews) [reply docs/tutorial/part-four/index.md]: yeah

Comment (ryanditjia) [reply docs/tutorial/part-four/index.md]: I’ve forgotten this is the style Gatsby prefers. On it.

**Comment by ryanditjia** [comment]
Wait until I’m done with part six and seven before merging. Thanks!

**Comment by ryanditjia** [comment]
Alright, done.

Side note: running format-markdown results in 20+ files right now. Perhaps it’s time to commit these, before it gets out of hand?

**Comment by ryanditjia** [comment]
I just re-read your [comment in the issue](https://github.com/gatsbyjs/gatsby/issues/5607#issuecomment-393907141).

I’ve been using className instead of css prop, would you like this changed to css instead?

**Comment by KyleAMathews** [comment]
No... I think I changed my mind. It'd be better to keep things explicit so it's less confusing for new people. If someone adopts Emotion, they'll probably learn about the css prop sooner or later.

File: docs/tutorial/part-seven/index.md [hunk 3]
```diff
@@ -317,39 +325,56 @@ The last step is to link to your new pages from the index page.
...
 import React from "react"
-import g from "glamorous"
+import { css } from "react-emotion"
 import { Link } from "gatsby"
-
 import { rhythm } from "../utils/typography"
+import Layout from "../components/layout"
 
 export default ({ data }) => {
+  console.log(data)
```
Comment (pieh) [inline docs/tutorial/part-seven/index.md]: Is this intentional?

Comment (ryanditjia) [reply docs/tutorial/part-seven/index.md]: Good catch. This was a line from part-six that I missed to remove. Fixed.

**Comment by ryanditjia** [comment]
I can send a PR, but I’m not sure whether to do master’s which will then be merged to v2, or to just do v2’s.

I’ve got a question I’ve been meaning to ask. What’s going to happen to v1’s docs, tutorials, package READMEs once v2 is the new default? Seeing how huge the adoption of Gatsby v1 has been, there’s going to be a lot of people who don’t have the resources to upgrade to v2 still needing to refer to v1’s docs. There could be merit to keeping, say, v1.gatsbyjs.org around.

**Pull request merged**